### DISCUSSION TITLE
No decorators in generated files

### DISCUSSION THREAD

**Discussion by ksharplee** [MichalLytek/type-graphql#1225] [Q&A]
I followed the installation guide in https://prisma.typegraphql.com/, after generating type files using prisma generate command, there aren't any decorators in them, any ideas?

I've enabled 'experimentalDecorators', 'emitDecoratorMetadata' in the tsconfig.json file, installed 'reflect-metadata', 'tslib'... etc, where did I go wrong

**Comment by MichalLytek** [comment]
Read the docs carefully. It's about the output setting and transpilling.
